File: app/src/main/java/com/arjanvlek/oxygenupdater/views/HelpActivity.java
package com.arjanvlek.oxygenupdater.views;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.MenuItem;

import com.arjanvlek.oxygenupdater.R;

public class HelpActivity extends AppCompatActivity {

    @Override
    public void onCreate(Bundle savedInstanceSate) {
        super.onCreate(savedInstanceSate);
        setContentView(R.layout.fragment_setup_2);
    }

    @Override
    public void onBackPressed() {
        finish();
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        switch (item.getItemId()) {
            // Respond to the action bar's Up/Home button
            case android.R.id.home:
                finish();
                return true;
        }
        return super.onOptionsItemSelected(item);
    }

}

File: app/src/main/java/com/arjanvlek/oxygenupdater/activities/HelpActivity.kt
package com.arjanvlek.oxygenupdater.activities

import android.os.Bundle
import android.view.MenuItem
import com.arjanvlek.oxygenupdater.R

class HelpActivity : SupportActionBarActivity() {

    public override fun onCreate(
        savedInstanceSate: Bundle?
    ) = super.onCreate(savedInstanceSate).also {
        setContentView(R.layout.activity_help)
    }

    override fun onBackPressed() = finish()

    /**
     * Respond to the action bar's Up/Home button.
     * Delegate to [onBackPressed] if [android.R.id.home] is clicked, otherwise call `super`
     */
    override fun onOptionsItemSelected(item: MenuItem) = when (item.itemId) {
        android.R.id.home -> onBackPressed().let { true }
        else -> super.onOptionsItemSelected(item)
    }
}
